src/duke/Deadline.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * This is a child class of duke.Task, duke.Deadline.
+ * duke.Deadline accepts another variable, 'by' that
+ * stores the time this duke.Deadline has to be completed
+ *
+ * @author  Hsiao Jiet
+ * @version 1.0
+ * @since   2022-1-15
+ */
+
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
Maybe it would be better to name this `deadline`, and maybe the class as `DeadlineTask`. What do you think?